MANIFEST.in

```diff
@@ -0,0 +1,8 @@
+graft src/python/grpcio/grpc
```

Comment:
It may be possible to do this such that the tools don't *need* to be made aware of any changes (through the magic of customized/redirected distribution utility commands). Looking into that instead of the previous instead.